### PR TITLE
CHE-2813 update style for popups in Dashboard

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@codenvy.com>

### What does this PR do?
Update style for popups in Dashboard
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/2813

selection_001
selection_002
selection_003
selection_006
selection_007
selection_008

#### Changelog
Update style for popups.

**Release notes**: 
- [x] Updated the styles for popups by adding header and making content look common

### CODE REVIEW

**Review by akurinnoy — APPROVED** [review]

**Comment by ashumilova** [comment]
@olexii4 please check the docs on modified popups - to be sure docs contain the same one.

**Comment by olexii4** [comment]
@ashumilova  the docs doesn't contain these popups

File: dashboard/src/app/administration/docker-registry/docker-registry-list/edit-registry/edit-registry.controller.ts [hunk 1]
```diff
@@ -9,57 +9,61 @@
...
+          this.cheNotification.showError(error.data && error.data.message ? error.data.message : 'Error.');
         });
       } else {
-        this.cheNotification.showInfo('Registry successfully edited.');
+        this.$mdDialog.hide();
       }
-    }, (error) => {
-      this.cheNotification.showError(error.data && error.data.message ? error.data.message : 'Edit registry error.');
+    }, (error: any) => {
+      this.cheNotification.showError(error.data && error.data.message ? error.data.message : 'Error.');
```
Comment (ashumilova) [inline dashboard/src/app/administration/docker-registry/docker-registry-list/edit-registry/edit-registry.controller.ts]: 'Error.' - add more description

Comment (olexii4) [reply dashboard/src/app/administration/docker-registry/docker-registry-list/edit-registry/edit-registry.controller.ts]: done

**Comment by ashumilova** [comment]
Following dialogs also should be fixed:

components
env_variables
machine
projects

**Comment by olexii4** [comment]
selection_024
selection_023
selection_022
selection_021
selection_020
Done

**Review by ashumilova — APPROVED** [review]

**Review by slemeur — APPROVED** [review]

**Pull request merged**